.github/workflows/ibis-main.yml

```diff
@@ -0,0 +1,206 @@
+# vim: filetype=yaml
+name: Ibis
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test_no_backends:
+    name: Test ${{ matrix.os }} python-${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+        python-version:
```

Comment:
This should include 3.8, right? There is a job running windows-py38, but it's actually showing up in the `Backends` workflow instead of this one.

Reply:
And possibly 3.10 too. Though I'm not sure whether we need the extensive OS parametrization too.

Reply:
We already have 48 builds which seems to be a bit overwhelming.

Reply:
:shrug: The number of builds that is overwhelming will vary with people. The builds finish relatively quickly given the extent of testing being done. I get what you're saying but I guess I'm more inclined to add tooling (hopefully in the form of a GitHub action, e.g., the test report) to make large build matrices manageable as opposed to cutting out tests to do so.

Reply:
FWIW, Cutting out 3.8 is probably fine, unless there are concerns about 3.8 specific bugs that would show up in our builds.

Reply:
My comment here was also about the fact that there is a 3.8 build for this set of parameters, but it's showing up in *Backends* in the UI and not in the *Ibis* workflow.

Reply:
That build seems to belong to the HDF5 backend, fixing.